src/Components/CreateImageWizard/steps/googleCloud.js

```diff
@@ -101,17 +101,26 @@ export default {
       component: componentTypes.PLAIN_TEXT,
       name: 'google-cloud-text-component',
       label: (
-        <Text>
-          Your image will be uploaded to Google Cloud Platform and shared with
-          the account you provide below. <br />
-          The shared image will expire within 14 days. To keep the image longer,
-          copy it to your Google Cloud Platform account.
-        </Text>
+        <p>
+          Your image will be uploaded to GCP and shared with the account you
+          provide below.
+        </p>
+      ),
+    },
+    {
+      component: componentTypes.PLAIN_TEXT,
+      name: 'google-cloud-text-component',
+      label: (
+        <p>
+          <b>The shared image will expire within 14 days.</b> To permanently
+          access the image, copy it to <br /> your Google Cloud Platform
```

Comment:
We don't need the `<br />` here, the text will automatically adjust to the width of the modal. :)

Reply:
thank you @regexowl if I remove the <br/> it look like that 
<img width="1103" alt="Screenshot 2023-06-27 at 15 01 06" src="https://github.com/RedHatInsights/image-builder-frontend/assets/73419853/c498d252-5d96-481c-9592-b9738c29ef39">
and its different from Skretch, is that ok?